libpod/image/image.go

```diff
@@ -1514,6 +1514,23 @@ func (i *Image) Save(ctx context.Context, source, format, output string, moreTag
 	return nil
 }
 
+// dockerArchiveDestReference returns a NamedTagged reference for a tagged image and nil for untagged image.
+func dockerArchiveDstReference(normalizedInput string) (reference.NamedTagged, error) {
+	if normalizedInput == "" {
+		return nil, errors.Errorf("docker-archive reference %s isn't of the form <path>[:<reference>]", normalizedInput)
```

Comment:
This previously didn’t fail, but just called `ParseReference` without a name = called `NewReference(output, nil)`.

(And if it were intentional, the error text does not match the situation, `normalizedInput` from `imageNameForSaveDestination` does not contain any path.)

Reply:
Update to not failing and instead `NewReference(output, nil)`. PTAL